website_sale_affiliate_portal/views/website_portal_sale_templates.xml

```diff
@@ -4,7 +4,7 @@
   <template
         id="portal_my_home_affiliations"
         name="Show Affiliations"
-        inherit_id="sale.portal_my_home_sale"
+        inherit_id="portal.portal_my_home"
```

Comment:
This is arguable, I would not do this in the migration. The `sale` module adds its own portal.portal_docs_entry sections to the o_portal_docs div, so we want to add affiliations after them probably, which is more explicit if you inherit the `sale.portal_my_home_sale` template, even if we may get the same result thanks to module load ordering.